src/list-inactive.ts

```diff
@@ -0,0 +1,186 @@
+import { AutoScalingClient, DescribeScalingActivitiesCommand } from "@aws-sdk/client-auto-scaling";
+import { CloudFormationClient, DescribeStackResourceCommand } from "@aws-sdk/client-cloudformation";
+import { GetFunctionCommand, LambdaClient } from "@aws-sdk/client-lambda";
+import { fromIni } from "@aws-sdk/credential-provider-ini";
+import { getStacks } from "./cloudformation";
+import { Config } from "./config";
+import type { StackMetadata } from "./types";
+
+const Inactive = {
+  NoRecentUpdate: "Stack has not been updated at all in the last 2 years.",
+  NoRecentAsgDeployment: "Stack ASG has not been deployed to in the last month.",
+  NoRecentLambdaDeployment: "Stack Lambda has not been updated in the last 6 months.",
+  TestInName: "Stack name contains 'Test', which suggests it may be temporary.",
+};
+
+type InactiveStatus = keyof typeof Inactive;
+
+const marshallTxt = (profile: string, region: string, stacks: Map<StackMetadata, InactiveStatus>): string => {
```

Comment:
My thinking here is to support a `format=csv|text` flag for all commands. But I haven't implemented CSV here yet.